aci/resource_aci_vmmvswitchpolicycont.go

```diff
@@ -0,0 +1,433 @@
+package aci
+
+import (
+	"fmt"
+	"log"
+
+	"github.com/ciscoecosystem/aci-go-client/client"
+	"github.com/ciscoecosystem/aci-go-client/models"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
+)
+
+func resourceAciVSwitchPolicyGroup() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceAciVSwitchPolicyGroupCreate,
+		Update: resourceAciVSwitchPolicyGroupUpdate,
+		Read:   resourceAciVSwitchPolicyGroupRead,
+		Delete: resourceAciVSwitchPolicyGroupDelete,
+
+		Importer: &schema.ResourceImporter{
+			State: resourceAciVSwitchPolicyGroupImport,
+		},
+
+		SchemaVersion: 1,
+
+		Schema: AppendBaseAttrSchema(map[string]*schema.Schema{
+			"vmm_domain_dn": &schema.Schema{
+				Type:     schema.TypeString,
+				Required: true,
+			},
+
+			"annotation": &schema.Schema{
+				Type: schema.TypeString,
+
+				Optional:    true,
+				Computed:    true,
+				Description: "Mo doc not defined in techpub!!!",
+			},
```

Comment:
Annotation field is not needed here, because I added annotation in the base attribute schema.